stdlib/config-api/src/test/resources/datafiles/default/ballerina.conf

```diff
@@ -52,3 +52,11 @@ World!!!
 [listenerConfig.keyStore]
 paths=["/etc", "/tmp", "/usr/lib/"]
 ports=[9090, 9091, 9092]
+
+[[foo]]
+a=1
+b=2
+
+[[foo]]
+a=3
+b=4
```

Comment:
Shall we add test cases for negative cases and multi dimensional arrays as well? See https://github.com/toml-lang/toml/tree/v0.4.0#array

Also a bit more complex case for arrays of table and a negative case as well: https://github.com/toml-lang/toml/tree/v0.4.0#array-of-tables

Reply:
Added more tests.